django/db/backends/mysql/operations.py

```diff
@@ -142,6 +143,8 @@ def adapt_datetimefield_value(self, value):
             return None
 
         # MySQL doesn't support tz-aware datetimes
+        if isinstance(value, Combinable):
```

Comment:
It looks like this new clause (and similar ones below) are splitting a comment from the code it applies to.